test/v2-migration-tests/src/test/resources/software/amazon/awssdk/v2migrationtests/maven-nocompile/after/src/main/java/foo/bar/S3Transforms.java

```diff
@@ -33,6 +35,12 @@
 
 public class S3Transforms {
 
+    void s3Pojos() {
+        DeleteObjectRequest deleteVersionRequest =
+            DeleteObjectRequest.builder().bucket("bucket").key("key").versionId("id").mfa(new MultiFactorAuthentication("serialNum", "token"))
```

Comment:
Can we add a comment to show how customers can fix it?

Reply:
Sure, will do as follow up. Want to merge these changes to include in the benchmarks